avocado/plugins/run.py

```diff
@@ -295,7 +295,7 @@ def run(self, config):
                 sys.exit(exit_codes.AVOCADO_FAIL)
 
         try:
-            suite = TestSuite.from_config(config, name='suite01')
+            suite = TestSuite.from_config(config, name='')
```

Comment:
This is exactly what I was imagining. In fact using default `suite01` has additional drawbacks like assuming the number and formatting of test suites, i.e. it adds a lot of extra assumption on best naming and format for everyone.